components/style/properties/gecko.mako.rs

```diff
@@ -392,6 +392,20 @@ impl ${style_struct.gecko_struct_name} {
     }
 </%def>
 
+<%def name="impl_simple_wrapper_setter(ident, gecko_ffi_name)">
+    #[allow(non_snake_case)]
+    pub fn set_${ident}(&mut self, v: longhands::${ident}::computed_value::T) {
+        ${set_gecko_property(gecko_ffi_name, "v.0")}
```

Comment:
We have something to implement stuff with conversions. Perhaps we should implement `From` and remove this mako code instead, wdyt?